library/formsoptions.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * Functions to be used in the forms.
```

Comment:
I get the clinical/visit/encounter forms change but not sure about what you want here or do you mean this for the function itself?